app/assets/javascripts/admin/addon/routes/admin-customize-colors.js

```diff
@@ -16,6 +16,14 @@ export default class AdminCustomizeColorsRoute extends Route {
   setupController(controller, model) {
     super.setupController(controller, model);
     controller.set("model", model.colorSchemes);
-    controller.set("defaultTheme", model.themes.findBy("default", true));
+
+    const themes = model.themes || [];
+    const defaultTheme = themes.findBy ? themes.findBy("default", true) : null;
```

Comment:
we're in the process of removing ember array mixin uses, so this ideally should be written w/o findBy